package.json

```diff
@@ -45,22 +48,25 @@
     "@pinia-plugin-persistedstate/nuxt": "^1.1.1",
     "@pinia/nuxt": "^0.5.0",
     "adm-zip": "~0.5.10",
-    "bcryptjs": "^2.4.3",
     "exceljs": "~4.3.0",
     "formidable": "~3.5.1",
     "into-stream": "~8.0.0",
     "jsonwebtoken": "~9.0.2",
     "ldapjs": "^3.0.7",
     "libxmljs": "^1.0.11",
     "luxon": "~3.4.3",
+    "mustache": "^4.2.0",
     "pg": "~8.11.3",
     "pg-hstore": "~2.3.4",
     "pinia": "^2.1.7",
     "postgres": "~3.3.5",
     "sequelize": "~6.32.1",
     "sqlite3": "~5.1.6",
+    "tar": "^6.2.0",
+    "ts-node": "^10.9.2",
```

Comment:
I thought we were getting rid of ts-node?  If it's no longer explicitly used by us then we can rid of it here I think.